From 4d88788a6efc68acf243a1a33d854f76b2f26dfd Mon Sep 17 00:00:00 2001
From: Joe Sapp <joe@babyjoe>
Date: Tue, 3 Nov 2015 21:23:05 -0500
Subject: [PATCH] Use Image.tobytes() instead of tostring()

tostring() was deprecated (in Pillow at least) in version 2.0.  See https://github.com/python-pillow/Pillow/commit/baa5143394708704328dcd46b0387f36a276a762
---
 gourmet/gtk_extras/ratingWidget.py             | 2 +-
 gourmet/plugins/browse_recipes/icon_helpers.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/gourmet/gtk_extras/ratingWidget.py b/gourmet/gtk_extras/ratingWidget.py
index 0e01735..efa6463 100644
--- a/gourmet/gtk_extras/ratingWidget.py
+++ b/gourmet/gtk_extras/ratingWidget.py
@@ -135,7 +135,7 @@ def get_pixbuf_from_image (self, image, make_white_opaque=True):
         if is_rgba: rowstride = 4
         else: rowstride = 3
         pb=gtk.gdk.pixbuf_new_from_data(
-            image.tostring(),
+            image.tobytes(),
             gtk.gdk.COLORSPACE_RGB,
             is_rgba,
             8,
diff --git a/gourmet/plugins/browse_recipes/icon_helpers.py b/gourmet/plugins/browse_recipes/icon_helpers.py
index 61c772c..2e7b08b 100644
--- a/gourmet/plugins/browse_recipes/icon_helpers.py
+++ b/gourmet/plugins/browse_recipes/icon_helpers.py
@@ -38,7 +38,7 @@ def get_pixbuf_from_image (image):
     if is_rgba: rowstride = 4
     else: rowstride = 3
     pb=gtk.gdk.pixbuf_new_from_data(
-        image.tostring(),
+        image.tobytes(),
         gtk.gdk.COLORSPACE_RGB,
         is_rgba,
         8,
